external_auth/views.py

```diff
@@ -49,13 +50,15 @@ def pakwheels_registration(request):
             user.profile.city = pakwheels_profile.city
             user.profile.country = pakwheels_profile.country
             user.save()
+            login(request, user)
```

Comment:
you still forgot to mark user as active: https://docs.djangoproject.com/en/1.11/ref/contrib/auth/#django.contrib.auth.models.User.is_active